### DISCUSSION TITLE
Rustic: A performant Rust implementation of Restic

### DISCUSSION THREAD

**Discussion by gerardbosch** [erikw/restic-automatic-backup-scheduler#101] [Ideas]
Happy to greet you!

I recently found this, I report it here just for having it on the radar :)

I'm quite happy with Restic for now, but I recently found a Rust implementation of Restic that seems to offer better performance, and who knows if could be a preferred choice in the future:

https://github.com/rustic-rs/rustic

This is based on Restic repo format, but I don't know it is feature complete (comparing to Restic) as it seems to be still in beta. But, what I've noticed is that supports ignoring backup based on `.gitignore`  –something that is not yet implemented in Restic, see https://github.com/restic/restic/issues/1514

And here the additional features: https://github.com/rustic-rs/rustic#comparison-with-restic

Have a nice day! ☀

**Comment by erikw** [comment]
Great find, def. something to keep on the radar 📡
